portal-web/test/functional/com/liferay/portalweb/macros/FormPortletConfiguration.macro

```diff
@@ -2,6 +2,15 @@ definition {
 
 	var formName = "Created Form Name";
 
+	macro changeLanguageTo {
+		Click(
+			changeLanguageTo = "${getLanguage}",
+			locator1 = "FormPortletConfiguration#SELECT_LANGUAGE");
+		Click(
+			changeLanguageTo = "${changeLanguageTo}",
```

Comment:
@wesleyroberts, please rename:

- `changeLanguageTo` to `key_language`
- `${changeLanguageTo}` to `${newLanguage}`

Reply:
changes made